lib/wallet/http.js

```diff
@@ -1303,6 +1303,21 @@ class HTTP extends Server {
         this.to('w:*', event, wallet.id, json);
     };
 
```

Comment:
We could probably use `handleTX` instead of creating a new function here, but the difference is in `this.to`

Reply:
`handleTX` doesn't pass along enough arguments, we need to add another method or refactor `handleTX`. The problem is that `handleTX` does nothing with the `tx` argument, and the covenant related events pass along the argument at that position. A new function called `handleCovenant` was created.